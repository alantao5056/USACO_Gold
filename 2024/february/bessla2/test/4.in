500 1000 20 10000 2
1 2 10000
2 3 10000
3 4 10000
4 5 10000
5 6 10000
6 7 10000
7 8 10000
8 9 10000
9 10 10000
10 11 10000
11 12 10000
12 13 10000
13 14 10000
14 15 10000
15 16 10000
16 17 10000
17 18 10000
18 19 10000
19 20 10000
20 21 10000
21 22 10000
22 23 10000
23 24 10000
24 25 10000
25 26 10000
26 27 10000
27 28 10000
28 29 10000
29 30 10000
30 31 10000
31 32 10000
32 33 10000
33 34 10000
34 35 10000
35 36 10000
36 37 10000
37 38 10000
38 39 10000
39 40 10000
40 41 10000
41 42 10000
42 43 10000
43 44 10000
44 45 10000
45 46 10000
46 47 10000
47 48 10000
48 49 10000
49 50 10000
50 51 10000
51 52 10000
52 53 10000
53 54 10000
54 55 10000
55 56 10000
56 57 10000
57 58 10000
58 59 10000
59 60 10000
60 61 10000
61 62 10000
62 63 10000
63 64 10000
64 65 10000
65 66 10000
66 67 10000
67 68 10000
68 69 10000
69 70 10000
70 71 10000
71 72 10000
72 73 10000
73 74 10000
74 75 10000
75 76 10000
76 77 10000
77 78 10000
78 79 10000
79 80 10000
80 81 10000
81 82 10000
82 83 10000
83 84 10000
84 85 10000
85 86 10000
86 87 10000
87 88 10000
88 89 10000
89 90 10000
90 91 10000
91 92 10000
92 93 10000
93 94 10000
94 95 10000
95 96 10000
96 97 10000
97 98 10000
98 99 10000
99 100 10000
100 101 10000
101 102 10000
102 103 10000
103 104 10000
104 105 10000
105 106 10000
106 107 10000
107 108 10000
108 109 10000
109 110 10000
110 111 10000
111 112 10000
112 113 10000
113 114 10000
114 115 10000
115 116 10000
116 117 10000
117 118 10000
118 119 10000
119 120 10000
120 121 10000
121 122 10000
122 123 10000
123 124 10000
124 125 10000
125 126 10000
126 127 10000
127 128 10000
128 129 10000
129 130 10000
130 131 10000
131 132 10000
132 133 10000
133 134 10000
134 135 10000
135 136 10000
136 137 10000
137 138 10000
138 139 10000
139 140 10000
140 141 10000
141 142 10000
142 143 10000
143 144 10000
144 145 10000
145 146 10000
146 147 10000
147 148 10000
148 149 10000
149 150 10000
150 151 10000
151 152 10000
152 153 10000
153 154 10000
154 155 10000
155 156 10000
156 157 10000
157 158 10000
158 159 10000
159 160 10000
160 161 10000
161 162 10000
162 163 10000
163 164 10000
164 165 10000
165 166 10000
166 167 10000
167 168 10000
168 169 10000
169 170 10000
170 171 10000
171 172 10000
172 173 10000
173 174 10000
174 175 10000
175 176 10000
176 177 10000
177 178 10000
178 179 10000
179 180 10000
180 181 10000
181 182 10000
182 183 10000
183 184 10000
184 185 10000
185 186 10000
186 187 10000
187 188 10000
188 189 10000
189 190 10000
190 191 10000
191 192 10000
192 193 10000
193 194 10000
194 195 10000
195 196 10000
196 197 10000
197 198 10000
198 199 10000
199 200 10000
200 201 10000
201 202 10000
202 203 10000
203 204 10000
204 205 10000
205 206 10000
206 207 10000
207 208 10000
208 209 10000
209 210 10000
210 211 10000
211 212 10000
212 213 10000
213 214 10000
214 215 10000
215 216 10000
216 217 10000
217 218 10000
218 219 10000
219 220 10000
220 221 10000
221 222 10000
222 223 10000
223 224 10000
224 225 10000
225 226 10000
226 227 10000
227 228 10000
228 229 10000
229 230 10000
230 231 10000
231 232 10000
232 233 10000
233 234 10000
234 235 10000
235 236 10000
236 237 10000
237 238 10000
238 239 10000
239 240 10000
240 241 10000
241 242 10000
242 243 10000
243 244 10000
244 245 10000
245 246 10000
246 247 10000
247 248 10000
248 249 10000
249 250 10000
250 251 10000
251 252 10000
252 253 10000
253 254 10000
254 255 10000
255 256 10000
256 257 10000
257 258 10000
258 259 10000
259 260 10000
260 261 10000
261 262 10000
262 263 10000
263 264 10000
264 265 10000
265 266 10000
266 267 10000
267 268 10000
268 269 10000
269 270 10000
270 271 10000
271 272 10000
272 273 10000
273 274 10000
274 275 10000
275 276 10000
276 277 10000
277 278 10000
278 279 10000
279 280 10000
280 281 10000
281 282 10000
282 283 10000
283 284 10000
284 285 10000
285 286 10000
286 287 10000
287 288 10000
288 289 10000
289 290 10000
290 291 10000
291 292 10000
292 293 10000
293 294 10000
294 295 10000
295 296 10000
296 297 10000
297 298 10000
298 299 10000
299 300 10000
300 301 10000
301 302 10000
302 303 10000
303 304 10000
304 305 10000
305 306 10000
306 307 10000
307 308 10000
308 309 10000
309 310 10000
310 311 10000
311 312 10000
312 313 10000
313 314 10000
314 315 10000
315 316 10000
316 317 10000
317 318 10000
318 319 10000
319 320 10000
320 321 10000
321 322 10000
322 323 10000
323 324 10000
324 325 10000
325 326 10000
326 327 10000
327 328 10000
328 329 10000
329 330 10000
330 331 10000
331 332 10000
332 333 10000
333 334 10000
334 335 10000
335 336 10000
336 337 10000
337 338 10000
338 339 10000
339 340 10000
340 341 10000
341 342 10000
342 343 10000
343 344 10000
344 345 10000
345 346 10000
346 347 10000
347 348 10000
348 349 10000
349 350 10000
350 351 10000
351 352 10000
352 353 10000
353 354 10000
354 355 10000
355 356 10000
356 357 10000
357 358 10000
358 359 10000
359 360 10000
360 361 10000
361 362 10000
362 363 10000
363 364 10000
364 365 10000
365 366 10000
366 367 10000
367 368 10000
368 369 10000
369 370 10000
370 371 10000
371 372 10000
372 373 10000
373 374 10000
374 375 10000
375 376 10000
376 377 10000
377 378 10000
378 379 10000
379 380 10000
380 381 10000
381 382 10000
382 383 10000
383 384 10000
384 385 10000
385 386 10000
386 387 10000
387 388 10000
388 389 10000
389 390 10000
390 391 10000
391 392 10000
392 393 10000
393 394 10000
394 395 10000
395 396 10000
396 397 10000
397 398 10000
398 399 10000
399 400 10000
400 401 10000
401 402 10000
402 403 10000
403 404 10000
404 405 10000
405 406 10000
406 407 10000
407 408 10000
408 409 10000
409 410 10000
410 411 10000
411 412 10000
412 413 10000
413 414 10000
414 415 10000
415 416 10000
416 417 10000
417 418 10000
418 419 10000
419 420 10000
420 421 10000
421 422 10000
422 423 10000
423 424 10000
424 425 10000
425 426 10000
426 427 10000
427 428 10000
428 429 10000
429 430 10000
430 431 10000
431 432 10000
432 433 10000
433 434 10000
434 435 10000
435 436 10000
436 437 10000
437 438 10000
438 439 10000
439 440 10000
440 441 10000
441 442 10000
442 443 10000
443 444 10000
444 445 10000
445 446 10000
446 447 10000
447 448 10000
448 449 10000
449 450 10000
450 451 10000
451 452 10000
452 453 10000
453 454 10000
454 455 10000
455 456 10000
456 457 10000
457 458 10000
458 459 10000
459 460 10000
460 461 10000
461 462 10000
462 463 10000
463 464 10000
464 465 10000
465 466 10000
466 467 10000
467 468 10000
468 469 10000
469 470 10000
470 471 10000
471 472 10000
472 473 10000
473 474 10000
474 475 10000
475 476 10000
476 477 10000
477 478 10000
478 479 10000
479 480 10000
480 481 10000
481 482 10000
482 483 10000
483 484 10000
484 485 10000
485 486 10000
486 487 10000
487 488 10000
488 489 10000
489 490 10000
490 491 10000
491 492 10000
492 493 10000
493 494 10000
494 495 10000
495 496 10000
496 497 10000
497 498 10000
498 499 10000
499 500 10000
476 63 9652
268 121 6959
180 212 526
364 201 5065
247 355 1053
39 60 4700
474 149 290
48 97 5194
355 320 246
302 165 2135
37 496 736
124 44 3177
135 142 7190
26 238 514
187 365 6758
89 128 5020
160 480 3484
227 38 5810
53 449 6086
89 203 3660
463 164 4561
302 424 4410
88 471 9267
278 500 9302
372 248 2460
87 344 1944
64 331 7586
260 442 8794
195 476 2575
256 125 9264
221 457 7820
93 79 2709
499 337 5474
243 195 2530
431 132 5841
481 231 4741
239 287 9762
339 92 4902
362 180 6578
220 125 1908
446 3 5223
87 228 6562
473 395 6613
280 193 9209
132 85 621
377 491 9557
384 491 7825
68 249 8671
232 124 6659
433 79 1351
401 487 9525
365 334 966
131 13 1026
20 70 9104
377 405 9739
256 411 3754
219 290 4681
137 428 1050
55 291 3589
133 301 2543
258 61 6752
80 233 8816
453 203 227
59 7 2545
257 417 5412
491 33 3519
312 371 7582
349 34 2295
368 41 9037
295 33 5630
95 73 7430
481 407 8913
94 222 9336
228 363 6063
25 334 663
489 104 273
55 255 3053
167 32 5888
106 68 5896
465 56 3694
212 168 1897
315 245 4731
3 443 2711
194 357 5236
269 300 3453
46 221 2750
295 238 6921
71 247 7162
342 305 7024
89 166 4684
193 249 2823
315 434 3028
402 326 9542
192 98 253
445 259 7556
228 49 3354
131 159 7592
379 431 8204
128 413 9024
499 150 549
461 111 5174
162 213 6585
22 259 8145
120 35 9354
343 345 3403
365 382 7186
158 319 6601
67 254 4724
213 74 1886
265 487 465
390 417 4387
283 368 1203
338 231 1923
149 37 2967
76 190 4132
85 291 1515
191 99 1641
439 434 4828
424 334 1621
70 372 9006
309 225 7774
179 210 2617
74 201 2862
395 126 1761
101 164 5566
331 241 6352
371 248 3525
14 47 5471
58 463 1045
213 391 2931
406 91 3200
385 138 5232
170 126 5126
135 76 8757
328 345 7277
104 345 2589
233 302 7626
197 11 726
88 31 8158
349 239 7847
378 258 9541
86 141 9442
144 179 6858
22 208 5919
37 308 1718
283 58 4495
181 95 8199
276 1 4712
342 284 3033
479 115 5205
358 37 1787
462 428 8630
425 191 9893
186 190 8706
363 374 8562
173 70 5583
122 285 4206
118 196 2696
333 123 7892
435 100 7233
30 280 4666
406 143 6815
268 263 3164
144 236 3759
211 372 9147
435 390 2422
300 453 7780
496 245 6345
491 373 5602
151 426 6707
369 397 8687
275 291 9275
6 393 6504
388 353 5478
305 247 5457
97 116 9473
425 273 8997
361 350 3681
327 258 9902
104 246 8326
326 148 7624
229 480 5605
189 41 506
431 356 8314
300 15 8097
309 450 3379
402 305 8969
13 435 8790
333 342 4607
162 288 4790
158 9 5335
315 444 8361
329 242 9957
138 56 5494
370 177 5766
455 235 8676
367 496 6170
309 108 7095
161 320 7378
259 342 6496
13 42 8396
152 241 4345
206 10 9141
6 407 407
88 152 1895
256 418 4531
172 4 4567
60 51 2361
60 121 1123
330 479 6406
380 442 5505
402 190 7710
212 410 6841
449 18 9406
378 329 3359
413 376 2714
42 268 4417
395 194 5879
384 121 2828
38 250 2053
121 212 4465
495 256 4853
217 339 569
292 470 455
461 396 7745
36 422 5637
308 331 9386
318 347 6382
318 339 4181
308 229 2345
260 87 3280
487 74 4970
219 170 31
280 88 2128
166 203 1028
486 428 2143
251 491 3263
22 500 5518
130 343 9243
475 412 9071
164 90 7104
449 444 4645
198 147 6824
422 5 2782
179 136 5366
220 253 4276
69 250 3462
417 94 9500
487 91 8683
295 228 1017
6 270 6072
427 90 842
43 30 1844
269 281 9957
229 421 8488
68 498 509
138 224 446
329 213 7789
461 458 8566
42 356 6287
361 195 8640
294 455 4480
484 158 1060
289 165 5977
88 210 5869
129 279 4807
49 253 7597
155 302 2771
393 352 1327
191 330 9516
305 250 7363
382 244 163
318 400 7757
19 301 3374
96 131 493
75 146 9768
427 220 7057
456 448 2122
210 112 7667
340 391 3083
360 309 5176
50 235 4684
35 330 5207
475 284 7978
242 490 5772
66 42 1802
77 467 6254
142 392 8633
379 401 8293
442 209 9777
331 242 1576
466 324 4104
201 175 6653
217 250 2439
429 470 4577
346 211 409
304 483 9546
188 436 7683
126 301 9494
122 178 3565
304 484 7948
11 431 3103
162 116 9341
206 205 4063
258 349 7270
75 500 4664
107 455 3833
336 253 6617
350 333 4080
322 55 9328
440 441 834
263 406 2244
351 119 6363
176 477 5631
237 399 593
53 71 6102
262 436 2584
488 438 8678
235 124 3602
108 118 9045
334 435 9269
155 120 9148
71 320 6261
158 416 9127
232 367 1349
447 364 7555
335 62 1101
66 242 353
49 423 5120
16 297 8855
152 106 7100
357 449 5738
377 187 6367
449 92 5691
14 176 6188
207 44 9031
195 41 5035
172 85 2550
117 262 7910
343 166 9436
213 10 8101
137 322 5445
252 95 3920
316 451 1728
255 280 5724
499 153 8659
474 84 7115
360 349 1758
6 34 4682
300 185 8627
15 156 2708
37 55 2381
140 326 5294
242 51 5096
161 440 1776
203 392 6723
339 192 1886
194 351 452
198 234 1197
273 262 9304
466 27 556
395 402 198
65 428 491
404 384 3833
287 14 4147
371 309 5140
29 486 4672
170 10 85
119 384 1483
331 347 4571
213 211 9804
24 213 730
221 116 2211
344 402 2024
144 401 6207
101 199 7346
13 237 9308
248 123 986
184 60 8941
239 40 7514
40 133 7214
268 167 5561
25 236 9013
291 216 5571
439 192 2260
159 466 416
23 426 3666
26 102 222
193 276 9203
234 218 962
392 65 3430
443 65 4859
437 26 2756
286 223 6051
191 441 3378
159 339 5853
21 91 8331
238 497 9719
169 232 6882
48 223 70
328 394 6513
39 198 9851
361 427 1332
13 2 8264
276 21 5941
421 80 8484
163 309 1799
166 272 9623
80 355 731
290 313 1686
389 66 4449
240 224 9597
53 429 4627
353 1 2009
273 48 3775
456 396 9395
204 174 1600
422 5 9779
75 382 8382
94 262 4223
422 204 4052
283 454 6977
65 440 5998
40 304 6418
317 309 8214
423 460 5738
312 459 553
392 140 7262
381 38 1614
76 3 3668
22 12 9991
489 173 4147
300 17 4993
449 127 8184
1 144 7192
423 203 3868
253 342 3974
90 280 9059
347 485 2694
288 35 1244
470 373 6348
13 239 2655
275 346 190
388 208 8065
354 70 5282
305 193 8909
383 272 886
113 9 5683
372 496 4405
215 31 4706
299 250 3972
233 428 1890
258 371 6197
424 50 2406
347 352 2470
195 81 1627
462 291 3093
473 411 2744
139 142 6653
486 329 8752
359 33 18
80 106 2689
245 246 4949
441 392 579
424 103 719
310 316 6664
417 378 2395
195 212 7946
435 344 6383
104 111 942
174 325 1509
31 374 9210
497 409 8548
64 30 8545
73 304 8913
193 125 7077
54 339 5854
294 408 2868
95 79 7470
466 375 7230
494 368 363
252 324 8723
117 24 4845
112 335 3815
41 73 2126
357 325 3918
439 49 2731
203 52 5630
226 364 9280
105 380 9678
319 14 6434
390 203 3746
126 318 9622
126 460 9467
9 179 9442
41 255 8001
447 182 4552
381 362 4842
284 355 1404
277 467 4544
263 435 3103
