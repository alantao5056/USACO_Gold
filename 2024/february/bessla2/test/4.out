5
85
96
131
132
300
